src/http_rest_api_handler.cpp

```diff
@@ -415,7 +415,7 @@ Status HttpRestApiHandler::processInferKFSRequest(const HttpRequestComponents& r
 
     if (!reporter) {
         return StatusCode::OK;
-        // TODO fix after Mediapipe metrics implementation
+        // There is no request time metric for MediaPipe endpoints
```

Comment:
Is it planned for unary or we won't implement it for both unary/streaming?

Reply:
There is idea for further research based on packet timestamps. We could change automatic timestamping to be based on real system date time instead of current approach - starting from 0 and incrementing by 1. This way we could be sure that generated packet for serialization of timestamp X has taken now()-X time for processing. However, this would enable manual client timestamping to interfere and bloat the metrics with fake request times due to client-server time desynchronizations. (for example different time zones) @atobiszei @dtrawins